### ISSUE TITLE
ssh alias documentation : non-standard port usage example

### ISSUE THREAD

**Issue by jambox** [wp-cli/wp-cli#3256]
Loving the new aliases feature! So much of a time-saver.

Still learning my way around the command line/SSH, so this might only be confusing for to a certain class of "beginner" user...

I couldn't figure out how to setup and alias in my `wp-cli.yml` file to connect to a non-standard port, found something in the github discussions in which you decided not to offer a `port` key in the alias definition and ultimately that it should be passed in this format: `ssh: {host.ip.here}:{port}`. This confused me because I'm used to ssh'ing to non-standard ports with the `-p` flag, i.e.:
```
ssh user@12.34.56.789 -p12345
```

I think there should be something explicit in the docs saying something like, "If you're setting up an alias with a non-standard port, append the port to your host IP with a colon." with an example.

**Comment by danielbachhuber** [comment]
> I couldn't figure out how to setup and alias in my wp-cli.yml file to connect to a non-standard port, found something in the github discussions in which you decided not to offer a port key in the alias definition and ultimately that it should be passed in this format
>
> [...]
>
> I think there should be something explicit in the docs saying something like, "If you're setting up an alias with a non-standard port, append the port to your host IP with a colon." with an example.

Where did you look for this information? Did you review the [config](http://wp-cli.org/config/) page by chance? I'm curious where you think might be the best place to put this information.

**Comment by jambox** [comment]
I first saw the aliases on the blog in the v0.24.0 release. Then I referenced the config page for a syntax refresher. But in terms of figuring out the port issue, I just dug through some of closed issues up here and found the discussion around non-standard ports which was sometime in May I think. 

I'd say a comment in the .yml example on the bottom of the config page would be sufficient, but I also think it'd be helpful to have a link to the aliases setup anywhere the global `--ssh` flag is described. And that, of course, could ultimately link to the bottom of the config page again.
